plugins/ubuntu/readme.md

```diff
@@ -1,21 +1,50 @@
-This plugin was created because the aliases in the debian plugin are inconsistent and hard to remember. Also this apt-priority detection that switched between apt-get and aptitude was dropped to keep it simpler. This plugin uses apt-get for everything but a few things that are only possible with aptitude I guess. Ubuntu does not have aptitude installed by default.
+# Ubuntu plugin
+
+This plugin adds completions and aliases for [Ubuntu](https://www.ubuntu.com/).
 
-acs = Apt-Cache Search  
-acp = Apt-Cache Policy
+To use it, add `ubuntu` to the plugins array in your zshrc file: 
+
+```zsh
+plugins=(... ubuntu)
+```
+### Note: 
```

Comment:
This note is no longer valid since we started preferring apt. You can say instead that commands that use `$APT` will automatically choose apt if installed, or defer to apt-get otherwise. This requires changing some of the aliases' commands in the aliases table (just use `$APT` there, people will get it).